docs/howto.md

```diff
@@ -63,12 +69,12 @@ int main() {
         };
```

Comment:
If you will make new edits, can you remove unnecessary semicolon? It's confusing.

Reply:
Did I add that semi-colon...? The diff for the PR doesn't seem to indicate that was changed in this PR.

Reply:
Of course, not. I think it is an improvement too. And I do not ask you to update your PR only for this change. But if other changes are required, it is good idea (not requirement) to apply this change.

Reply:
@jonesmz , @bmi-x  this PR relates to the part which has significant changes in #171, and it is better to improve the #171 since @koshachy is one who responsible for the future of the documentation.

Reply:
Hi @jonesmz , thanks for the PR!

There's a lot of changes in my and your PRs and conflicts are inevident. For now, I revise the how-to section at all, so I guess it's better to merge your changes.

I'll review your PR in a few days. And then we can merge it to master.

Sorry for any inconvenience.

Reply:
@koshachy Ok, sounds good.